custom_components/smart_irrigation/__init__.py

```diff
@@ -1248,20 +1260,22 @@ async def calculate_module(self, zone, weatherdata, forecastdata):
                     const.UNIT_INCH, const.UNIT_MM, drainage_rate
                 )
             _LOGGER.debug(f"[calculate-module]: drainage_rate: {drainage_rate}")
-            newbucket = (
-                bucket + (delta * hour_multiplier) - (drainage_rate * hour_multiplier)
-            )
+            newbucket = bucket + (delta * hour_multiplier)
```

Comment:
this seems to be a duplicate from line L1240?

Reply:
oops, copy/paste error.